Documentation/backward_compatibility.md

```diff
@@ -27,7 +29,7 @@ https://github.com/coreos/etcd/blob/master/Documentation/configuration.md.
 
 #### Standby
 
-etcd 0.4’s standby mode has been deprecated by 2.0’s [proxy mode][proxymode]. 
+etcd 0.4’s standby mode has been deprecated by 2.0’s [proxy mode][proxymode].
```

Comment:
etcd 0.4’s standby mode has been deprecated. proxy mode is introduced to solve a subset of problems standby was solving.